starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -219,7 +214,11 @@ async def test_warning_when_max_fee_equals_to_zero(map_contract):
         DeprecationWarning,
         match=r"Transaction will fail with max_fee set to 0. Change it to a higher value.",
     ):
-        await map_contract.functions["put"].invoke(10, 20, max_fee=0)
+        # try except have to be added because when running on integration it will throw an error (max_fee=0)
```

Comment:
What happens on other envs?

Reply:
on testnet result is the same, but devnet accepts transactions with max_fee=0

Reply:
Let's change "when running on integration" to "when running on a real environment"